consumer-configuration/README.adoc

```diff
@@ -291,18 +292,16 @@ Alternatively, select a consumer group for a specific topic.
 --
 . In the *Consumer groups* page, select the options icon (three vertical dots) for the relevant consumer group and click *Reset Offset*.
 . Select a topic.
-. Choose a new offset position from *Latest*, *Earliest*, or *Custom offset*.
+. Choose a new offset position from *Absolute*, *Latest*, or *Earliest*.
```

Comment:
The CLI has an extra reset offset option - Timestamp. Will it be worth adding here, with a note maybe.
```
rhoas kafka consumer-group reset-offset --id consumer_group_1 --offset timestamp --value "2016-06-23T09:07:21-07:00"
```
It is supposed to be added to UI later.

Reply:
Yes. We might want to take it out when the UI reflects the option, but makes sense to add a note here.